README.md

```diff
@@ -13,7 +13,16 @@ yarn install
 npm install
 ```
 
-Next, add karma to your path if it isn't already.
+Next, run the tests using yarn or npm:
+
+```shell script
+yarn test
+# or
+npm test
+```
+
+## Running tests interactively
+You can also run the tests "interactively" via Karma directly. Add Karma to your path if it isn't already:
```

Comment:
When you call yarn test, it calls karma. Does that step require karma to be globally installed? If so, it might be better to move the karma path line above

Reply:
Nope, karma-cli is not required to be globally installed when run via yarn or npm. I have tested this on a machine where I don't have it installed, and both "yarn test" and "npm test" work as expected.

Thanks for bringing this up though!